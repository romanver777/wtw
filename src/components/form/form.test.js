/* eslint-disable @typescript-eslint/no-unsafe-call */
/* eslint-disable @typescript-eslint/no-unsafe-member-access */
/* eslint-disable @typescript-eslint/no-unsafe-assignment */
import React from "react";
import TestRenderer from "react-test-renderer";
import Form from "./form";

describe("Form component", () => {
  const Trenderer = TestRenderer.create(<Form />);

  it("renders correctly", () => {
    expect(Trenderer.toJSON()).toMatchSnapshot();
  });
});
