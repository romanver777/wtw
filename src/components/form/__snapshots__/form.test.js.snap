// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component renders correctly 1`] = `
<form
  action="#"
  className="sign-in__form"
>
  <div
    className="sign-in__fields"
  >
    <div
      className="sign-in__field"
    >
      <input
        className="sign-in__input"
        id="user-email"
        name="user-email"
        placeholder="Email"
        type="email"
      />
      <label
        className="sign-in__label visually-hidden"
        htmlFor="user-email"
      >
        Email
      </label>
    </div>
    <div
      className="sign-in__field"
    >
      <input
        className="sign-in__input"
        id="user-password"
        name="user-password"
        placeholder="Пароль"
        type="password"
      />
      <label
        className="sign-in__label visually-hidden"
        htmlFor="user-password"
      >
        Пароль
      </label>
    </div>
  </div>
  <div
    className="sign-in__submit"
  >
    <button
      className="sign-in__btn"
      type="submit"
    >
      Войти
    </button>
  </div>
</form>
`;
